perfkitbenchmarker/openstack/os_disk.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import time
+
+from perfkitbenchmarker.openstack import utils as os_utils
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import disk
+from perfkitbenchmarker.openstack import utils
+from perfkitbenchmarker.openstack.utils import retry_authorization
+
+
+FLAGS = flags.FLAGS
+
+
+class OpenStackDisk(disk.BaseDisk):
+    characters = (c for c in "cdefghijklmnopqrstuvwxyz")
+
+    def __init__(self, disk_spec, name, zone, project, image=None):
+        super(OpenStackDisk, self).__init__(disk_spec)
+        self.__nclient = utils.NovaClient(FLAGS.os_auth_url,
+                                          FLAGS.os_tenant,
+                                          FLAGS.os_username,
+                                          FLAGS.os_passwd
+                                          )
+        self.attached_vm_name = None
+        self.attached_vm_id = -1
+        self.image = image
+        self.name = name
+        self.zone = zone
+        self.project = project
+        self.device = ""
+
+    def _Create(self):
+        self._disk = self.__nclient.volumes.create(self.disk_size,
+                                                   display_name=self.name,
+                                                   availability_zone=self.zone,
+                                                   imageRef=self.image,
+                                                   )
+        time.sleep(1)
+        is_unavailable = True
+        while is_unavailable:
```

Comment:
Move the `time.sleep(1)` from above into the loop here? Currently you're busy looping with no delay.